versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
+
+The serialization of the request body should be specified by the `consumes` attribute.
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="requestBodyDescription"></a>description | `string` | A brief description of the request body. This could contain examples of use.  [GFM syntax](https://help.github.com/articles/github-flavored-markdown) can be used for rich text representation.
+<a name="requestBodySchema"></a>schema | [Schema Object](#schemaObject) | The schema defining the type used for the request body.
+<a name="requestBodyExamples"></a>examples | [Examples Object](#examplesObject) | Examples of the request body, referenced by mime type.
+
+
+##### Patterned Fields
+Field Pattern | Type | Description
+---|:---:|---
+<a name="parameterExtensions"></a>^x- | Any | Allows extensions to the OpenAPI Schema. The field name MUST begin with `x-`, for example, `x-internal-id`. The value can be `null`, a primitive, an array or an object. See [Vendor Extensions](#vendorExtensions) for further details.
+
+
+##### Request Body Examples
+
+A request body with a referenced model definition.
+```js
+{
+  "description": "user to add to the system",
+  "schema": {
+    "$ref": "#/definitions/User"
+  },
+  "examples": {
```

Comment:
Looks like we can't avoid discussing how `examples` are written. I don't know what it means to reference an external non-json/yaml file. Also, `examples` values were intended to be taken as-is, so the `$ref` *is* the example in this case. Furthermore (and related to the two points above) - the only real way to represent non-json examples is to quote them as strings, but for JSON many people opt to just using JSON as-is without quoting. This makes processing a bit more difficult.

This is probably out of scope for this PR though and should be handled separately.